### PR TITLE
Add Danish Political Comments Dataset

### PR DESCRIPTION
**Pull request by abhishekkrthakur**

### CODE REVIEW

File: datasets/danish_political_comments/danish_political_comments.py [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+
+_DESCRIPTION = """\
+The dataset consists of 9008 sentences that are labelled with fine-grained polarity in the range from -2 to 2 (negative to postive). The quality of the fine-grained is not cross validated and is therefore subject to uncertainties; however, the simple polarity has been cross validated and therefore is considered to be more correct.
+"""
+_HOMEPAGE_URL = "https://github.com/steffan267/Sentiment-Analysis-on-Danish-Social-Media"
+_URL = (
+    "https://raw.githubusercontent.com/steffan267/Sentiment-Analysis-on-Danish-Social-Media/master/all_sentences.tsv"
+)
+_CITATION = "https://github.com/lucaspuvis/SAM/blob/master/Thesis.pdf"
```
Comment (lhoestq) [inline datasets/danish_political_comments/danish_political_comments.py:27]: Can you add the citation in the bibtex format ?
Or maybe it doesn't exist ?

File: datasets/danish_political_comments/danish_political_comments.py [hunk 2]
```diff
@@ -0,0 +1,72 @@
...
+    VERSION = datasets.Version("0.9.1")
+
+    def _info(self):
+        return datasets.DatasetInfo(
+            description=_DESCRIPTION,
+            features=datasets.Features(
+                {
+                    "id": datasets.Value("string"),
+                    "sentence": datasets.Value("string"),
+                    "target": datasets.Value("string"),
```
Comment (lhoestq) [inline datasets/danish_political_comments/danish_political_comments.py:40]: maybe use ClassLabel here ?

**Review by lhoestq — COMMENTED** [review]
Simple yet effective :)
thanks for adding this dataset !

I left a few comments.
Don't forget to add the tags (the yaml part at the top of the dataset card)

File: datasets/danish_political_comments/danish_political_comments.py (hunk 1 above)
Comment (abhishekkrthakur) [reply datasets/danish_political_comments/danish_political_comments.py:27]: It doesnt exist unfortunately.

File: datasets/danish_political_comments/danish_political_comments.py (hunk 2 above)
Comment (abhishekkrthakur) [reply datasets/danish_political_comments/danish_political_comments.py]: done

**Review by patrickvonplaten — APPROVED** [review]

**Review by lhoestq — APPROVED** [review]
thanks ! LGTM

**Pull request merged**